src/Components/ProjectCard.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import Chip from '@material-ui/core/Chip';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { useStyles } from './ProjectCardStyle';
+import RefreshRounded from '@material-ui/icons/RefreshRounded';
+import FileCopyRounded from '@material-ui/icons/FileCopyRounded';
+import DeleteRounded from '@material-ui/icons/DeleteRounded';
+
+export default function ProjectCard(props) {
+  const classes = useStyles();
+  const {
+    projectStatus,
+    statusTitle,
+    projectName,
+    updatedBy,
+    startedOn,
+    updatedOn,
+    completedOn,
+    handleDeleteClick,
+    handleCloneClick,
+    handleResumeClick,
+  } = props;
+  return (
+    <React.Fragment>
+      <Card className={classes.root} raised={true}>
+        <CardContent className={classes.cardContent}>
+          <div className={classes.statusRow}>
+            <Typography
+              className={
+                projectStatus ? classes.statusColor : classes.viewReports
+              }
+            >
+              {statusTitle}
+            </Typography>
+            <Chip
+              className={
+                projectStatus ? classes.chipColor : classes.designChipColor
+              }
+              label={projectStatus ? 'Deployment' : 'Design'}
+            />
+          </div>
+
+          <div className={classes.title}>{projectName}</div>
+          <div>Updated by: {updatedBy}</div>
+        </CardContent>
+        <CardActions>
+          <div className={classes.statusRow}>
```

Comment:
status row needs a bit more padding to align with the other text on the card: 
![image](https://user-images.githubusercontent.com/13375042/136995860-7d1f4d58-075d-471e-99a3-2afc04c65b16.png)